packages/grafana-toolkit/README.md

```diff
@@ -1,55 +1,82 @@
+- `-w`, `--watch` - run development task in a watch
 
-### Publishing to npm
-The publish process is now manual. Follow the steps to publish @grafana/toolkit to npm
-1. From Grafana root dir: `./node_modules/.bin/grafana-toolkit toolkit:build`
-2. `cd packages/grafana-toolkit/dist`
-3. Open `package.json`, change version according to current version on npm (https://www.npmjs.com/package/@grafana/toolkit)
-4. Run `npm publish --tag next` - for dev purposes we now publish on `next` channel
-
-Note, that for publishing you need to be part of Grafana npm org and you need to be logged in to npm in your terminal (`npm login`).
-
+### Testing extensions
+`plugin:test`
 
-## Grafana extensions development with grafana-toolkit overview
-### Available tasks
-#### `grafana-toolkit plugin:test`
-Runs Jest against your codebase. See [Tests](#tests) for more details.
+> Runs Jest against your codebase
 
 Available options:
-- `-u, --updateSnapshot` - performs snapshots update
-- `--coverage` - reports code coverage
 - `--watch` - runs tests in interactive watch mode
+- `--coverage` - reports code coverage
+- `-u`, `--updateSnapshot` - performs snapshots update
 - `--testNamePattern=<regex>` - runs test with names that match provided regex (https://jestjs.io/docs/en/cli#testnamepattern-regex)
 - `--testPathPattern=<regex>` - runs test with paths that match provided regex (https://jestjs.io/docs/en/cli#testpathpattern-regex)
 
-#### `grafana-toolkit plugin:dev`
-Compiles plugin in development mode.
+#### Configuring Enzyme
+We are not opinionated about tool used for implmenting tests. Internally at Grafana we use Enzyme. If you want to configure Enzyme as a testing utility, you need to configure `enzyme-adapter-react`. To do so, you need to create `[YOUR_EXTENSION]/config/jest-setup.ts` file that will provide React/Enzyme setup. Simply copy following code into that file to get Enzyme working with React:
 
-Available options:
-- `-w, --watch` - runs `plugin:dev` task in watch mode
-#### `grafana-toolkit plugin:build`
-Compiles plugin in production mode
+```ts
+import { configure } from 'enzyme';
+import Adapter from 'enzyme-adapter-react-16';
+
+configure({ adapter: new Adapter() });
+```
 
+grafana-toolkit will use that file as Jest's setup file.
+
+You can also setup Jest with shims of your needs by creating `jest-shim.ts` file in the same directory: `[YOUR_EXTENSION]/config/jest-shim.ts`
+
+Adidtionaly, you can provide Jest config via package.json file. For more details please refer to [Jest docs](https://jest-bot.github.io/jest/docs/configuration.html).
+
+We support following Jest config properties:
+- [`snapshotSerializers`](https://jest-bot.github.io/jest/docs/configuration.html#snapshotserializers-array-string)
+- [`moduleNameMapper`](https://jestjs.io/docs/en/configuration#modulenamemapper-object-string-string)
+
+
+### Building extensions
+
+`plugin:build`
+
+> Creates production ready build of your extension
+
+
+## grafana-toolkit tooling overview
```

Comment:
What mindset does the user have when reading this section? If they got this far into the README, I think we can assume it's because there's still something they want to be able to do. 

I'll let you decide on what you think the focus should be, but here are some ideas:

```
## FAQ

### How can I configure my plugin to use TSLint?
```

or ...

```
## Customization/How do I customize grafana-toolkit?

### TSLint

### Prettier
```